crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -814,3 +821,19 @@ fn remove_whitespace(s: &str) -> String {
 async fn publish_a_fake_jwt_token(broker: &MqttProcessHandler) {
     let _ = broker.publish("c8y/s/dat", "71,1111").await.unwrap();
 }
+
+pub async fn clear_session() -> Result<(), anyhow::Error> {
+    dbg!("Clear tedge sm mapper session");
+    let operations = Operations::try_new("/etc/tedge/operations", "c8y")?;
+    let topics = vec![
+        "tedge/commands/res/software/update",
+        "tedge/commands/res/software/list",
+        "tedge/commands/res/control/restart",
+    ];
+    let config = Config::default()
+        .with_session_name("c8y-mapper-test")
+        .with_clean_session(true)
+        .with_subscriptions(topics.try_into().unwrap());
```

Comment:
Neither the clean session flag nor the subscriptions are required when calling `mqtt_channel::clear_session`

Reply:
I removed this code because it did not work properly. I use keep_alive option for the testing purpose.